common/source/docs/common-downloading-and-analyzing-data-logs-in-mission-planner.rst

```diff
@@ -26,6 +28,7 @@ Logging Parameters
 
 Some commonly used parameters are:
 
+- :ref:`LOG_BACKEND_TYPE<LOG_BACKEND_TYPE>` : Bitmask for where to save logs to. The choices are: stream over MAVLink, save to SD card (if present) or save to dataflash chip (if present).
```

Comment:
not clear enough about which value does what...does not correlate to the order of values...please be more specific

Reply:
I've now given examples of common values